README.md

```diff
@@ -1,3 +1,6 @@
+# Brew Bot
+![Node CI](https://github.com/codedrunks/BrewBot/workflows/Node%20CI/badge.svg)
```

Comment:
does the badge make sense here if CI is triggered on pull request?

Reply:
it doesn't lol but i thought i'd add it now since we'll probably need it for when we got tests to run